### DISCUSSION TITLE
Blocking Chat & SMS spammers

### DISCUSSION THREAD

**Discussion by Yimura** [YimMenu/YimMenu#579] [Ideas]
1. Same message content being spammed (same content how often?)
2. Spam in general (frequency)
3. Filtering specific keywords

When filtering keywords I'd prefer using either hyper specific keywords or requiring at least 2 keywords before a message gets filtered out.

**Comment by CRGBS** [comment]
maybe we can custom block.txt
to custom filtering keywords

**Comment by iwanttodo1t** [comment]
Block messages directly from newly joined players

**Reply by iwanttodo1t** [reply]
Blacklist players who send a lot of messages just after joining the game

**Reply by iwanttodo1t** [reply]
However, it is useless to protect yourself alone, and others will not be able to see you from the large number of messages

**Reply by iwanttodo1t** [reply]
Kicking it out is probably the best way to go

**Comment by iwanttodo1t** [comment]
Block SMS messages from strangers

**Comment by iwanttodo1t** [comment]
Setting keywords is always one step slower
Kick out a player who sends multiple messages in a short period of time after joining a session

**Reply by iwanttodo1t** [reply]
Normal players don't send multiple messages as soon as they join a session

**Reply by iwanttodo1t** [reply]
Can this idea be done?

**Reply by shroomjaks** [reply]
Through lua yes
